exporter/stackdriver/stats_test.go

```diff
@@ -803,20 +766,3 @@ func newTestCumViewData(v *stats.View, start, end time.Time, data1, data2 stats.
 	}
 }
 
-func newTestDistViewData(v *stats.View, start, end time.Time) *stats.ViewData {
```

Comment:
Am not sure why we removed this helper? Aren't we still using Distribution aggregations in the tests?

Reply:
It was only used in one place and I removed that reference so I also removed the function. That place was basically asserting that we don't create SD time series for Interval windows, an assertion that no longer made sense after the current change. I agree that we should have a test for distribution, but it looks like we don't have this right now and I don't really have the context to write one as part of this PR. So I'm going to add back the function and just add a TODO to write a proper test for SD distribution.